packages/config/src/bridges/symbiosis.ts

```diff
@@ -0,0 +1,167 @@
+import { ProjectId, UnixTime } from '@l2beat/shared-pure'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { NUGGETS } from '../layer2s'
+import { RISK_VIEW } from './common'
+import { Bridge } from './types'
+
+const discovery = new ProjectDiscovery('symbiosis')
+
+export const symbiosis: Bridge = {
+  type: 'bridge',
+  id: ProjectId('symbiosis'),
+  display: {
+    name: 'Symbiosis',
+    slug: 'symbiosis',
+    category: 'Hybrid',
+    description:
+      'Symbiosis is a cross-chain AMM DEX that pools together liquidity from different networks, whether they use EVM technology or not.',
+    links: {
```

Comment:
maybe also link to the app?

Reply:
If you think it will be convenient to add it here, then: https://app.symbiosis.finance/